pkgs/tools/misc/zoxide/default.nix

```diff
@@ -28,7 +28,7 @@ rustPlatform.buildRustPackage rec {
       --replace '"fzf"' '"${fzf}/bin/fzf"'
   '';
 
-  cargoSha256 = "sha256-JRWlHwPFqNC/IIKQqRQszx4HHW1XsmROA67KmnxkmWQ=";
+  cargoSha256 = "sha256-1ptx6AE6Q47w2AP4f5xjYUg9EnWae+0NTZvFQZh9PV8=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/bq4w81f9q5dwdbvmnflcxfkaffas35dc-zoxide-0.9.3-vendor.tar.gz.drv':
         specified: sha256-1ptx6AE6Q47w2AP4f5xjYUg9EnWae+0NTZvFQZh9PV8=
            got:    sha256-fNAS2pCt3Oxz8qyn6GnJpx3Tq/EyPzbwns28tKeMqf0=
```

Reply:
But this isn't Darwin specific, is it?

Reply:
I'm not sure where I got the previous hash, but I recalculated it on x86_64-linux and aarch64-darwin and got this same hash. Updated.